RELEASENOTES.md

```diff
@@ -10,6 +10,7 @@ __API Changes__:
 
 __Bug Fixes__:
 
+- Now `DisposeScopes` could be correctly disposed in a wrong order.
```

Comment:
This comment doesn't make sense. If it's the wrong order, it's not correct.

Reply:
@NiklasGustafsson

Hmm... Sorry for my poor English. Could you please find a more accurate description for this?

Reply:
Now that I understand the reasoning behind it (seem my comments in the issue), I think:

'Allow dispose scopes to be disposed out of LIFO order.'